govcd/vapp.go

```diff
@@ -567,6 +568,31 @@ func (vapp *VApp) GetStatus() (string, error) {
 	return types.VAppStatuses[vapp.VApp.Status], nil
 }
 
+// StatusWaitNot blocks until the status of vApp is not equal to unwantedStatus.
```

Comment:
I believe you meant "blocks until the status of vApp _is_ equal to unwantedStatus"